eng/scripts/Language-Settings.ps1

```diff
@@ -3,6 +3,7 @@ $PackageRepository = "Maven"
 $packagePattern = "*.pom"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=java%2F&delimiter=%2F"
+$PACKAGE_INSTALL_NOTES_REGEX = "<artifactId>(?<PackageName>.*)<\/artifactId>\r\n\s+<version>(?<Version>.*)<\/version>"
```

Comment:
Should we do this as a function similar to how you are doing the install notes? That will give it the most flexibility.